builtin/providers/aws/provider.go

```diff
@@ -159,6 +159,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_s3_bucket_object":         dataSourceAwsS3BucketObject(),
 			"aws_subnet":                   dataSourceAwsSubnet(),
 			"aws_security_group":           dataSourceAwsSecurityGroup(),
+			"aws_volume":                   dataSourceAwsVolumes(),
```

Comment:
I'm not sure I can really explain why, but it feels weird to me not to name this `aws_ebs_volume`... I guess because it doesn't really feel "specific enough" without it, even though I'd concede that the naming scheme until now has generally been to use generic names for EC2 stuff.

A slightly stronger (but still rather weak) argument would be that when paired with `aws_ebs_snapshot` these would group together nicely in an alphabetical list.